### DISCUSSION TITLE
Add discussions state to repo REST endpoint

### DISCUSSION THREAD

**Discussion by dstrome** [community/community#11757] [Discussions]
Could you *please* add a boolean to the repo REST endpoint to indicate whether discussions are enabled on a repo? Please don't say use GraphQL. First, I just spent over an hour trying to even find how to get property info for a repo, and another hour trying to find property that is something like "discussionsEnabled" or something. I still don't know.

I don't understand why we're being pushed towards GraphQL which is orders of magnitude more complex than REST. With REST I can just submit a request to an endpoint and get back the data I need. One line, one operation. Picking apart the object is simple and logical. It's just JSON. GraphQL requires a 10 line query just to return a couple properties. 

The thing that really gets me is that some GraphQL operations require a REST call first to get info to make the GraphQL call. 

The REST implementation is simple and effective. Please continue to support it so we can focus on creating solutions rather than spending 3/4 of the time figuring out how to just make a query and parse the results.

**Comment by Marychoffel22** [comment]
Los Angeles California 90014

**Reply by Marychoffel22** [reply]
Boost

**Reply by Marychoffel22** [reply]
Danielchoffel299@smpt.com
